WordPress/src/main/java/org/wordpress/android/ui/prefs/SettingsActivity.java

```diff
@@ -26,61 +22,21 @@ public void onCreate(Bundle savedInstanceState) {
             actionBar.setHomeButtonEnabled(true);
             actionBar.setDisplayHomeAsUpEnabled(true);
         }
-        setContentView(R.layout.settings_activity);
 
         FragmentManager fragmentManager = getFragmentManager();
-        if (savedInstanceState == null) {
+        Fragment settingsFragment = fragmentManager.findFragmentByTag(KEY_SETTINGS_FRAGMENT);
+        Fragment passcodeFragment = fragmentManager.findFragmentByTag(KEY_PASSCODE_FRAGMENT);
+        if (settingsFragment == null || passcodeFragment == null) {
             Bundle passcodeArgs = new Bundle();
             passcodeArgs.putBoolean(PasscodePreferenceFragment.KEY_SHOULD_INFLATE, false);
-            mSettingsFragment = new SettingsFragment();
-            mPasscodePreferenceFragment = new PasscodePreferenceFragment();
-            mPasscodePreferenceFragment.setArguments(passcodeArgs);
+            settingsFragment = new SettingsFragment();
+            passcodeFragment = new PasscodePreferenceFragment();
+            passcodeFragment.setArguments(passcodeArgs);
 
             fragmentManager.beginTransaction()
-                    .add(R.id.fragment_container, mSettingsFragment)
-                    .add(R.id.fragment_container, mPasscodePreferenceFragment)
+                    .replace(android.R.id.content, passcodeFragment, KEY_PASSCODE_FRAGMENT)
```

Comment:
`replace` will remove any existing fragments from the specified container before calling `add`. There's some good info [here.](http://stackoverflow.com/questions/18634207/difference-between-add-replace-and-addtobackstack)